Documentation/network/kubernetes/bandwidth-manager.rst

```diff
@@ -255,7 +255,23 @@ which are in the initial network namespace (hostns). BBR also needs eBPF Host-Ro
 in order to retain the network packet's socket association all the way until the
 packet hits the FQ queueing discipline on the physical device in the host namespace.
 (Without eBPF Host-Routing the packet's socket association would otherwise be orphaned
-inside the host stacks forwarding/routing layer.)
+inside the host stacks forwarding/routing layer.). By default, Cilium will not start
```

Comment:
This text is in the section "BBR for Pods", but it seems focused on the host network namespace which is not just hostNetwork pods but all non-Pod traffic for the node.

I would suggest creating a new section something like "BBR for the host" and transfer the text there. I would also suggest referencing which configuration/kernel might be common to need such a workaround so that we can know when to deprecate/remove this option.